### PR TITLE
Add new Kubernetes procedure for single-node App Dev environment

### PR DESCRIPTION
**Pull request by chenette**

### CODE REVIEW

File: docs/source/app_developers_guide/kubernetes.rst [hunk 1]
```diff
@@ -0,0 +1,677 @@
...
+ * Starting Minikube
+ * Starting Sawtooth in a Kubernetes cluster
+ * Connecting to the Sawtooth shell container
+ * Using Sawtooth commands to submit transactions, display block data, and view
+   global state
+ * Examining Sawtooth logs
+ * Stopping Sawtooth and deleting the Kubernetes cluster
+
+After completing this procedure, you will have the environment required for
+for the other tutorials in this guide, :doc:`intro_xo_transaction_family` and
```
Comment (rberg2) [inline docs/source/app_developers_guide/kubernetes.rst]: extra for

File: docs/source/app_developers_guide/kubernetes.rst [hunk 2]
```diff
@@ -0,0 +1,677 @@
...
+(validator, REST API, and transaction processors) has both a debug log and an
+error log.
+
+For example, you can connect to the validator container and display the contents
+of ``/var/log/sawtooth``:
+
+.. code-block:: console
+
+   $ kubectl exec -it $(kubectl get pods | awk '/sawtooth-0/{print $1}') --container sawtooth-validator -- bash
+   root@podname#
```
Comment (rberg2) [inline docs/source/app_developers_guide/kubernetes.rst]: I like how we have the ls command here in the next example. What do you think of putting that here to? as is it looks like the logs would be in /

**Comment by chenette** [comment]
Reviewers:  The diagram in the HTML build artifact might appear to have font problems, but that's a build-artifact-only issue. Please look at the PDF (LaTeX) image instead of the SVG (HTML) version. 
https://build.sawtooth.me/job/Sawtooth-Hyperledger/job/sawtooth-core/job/PR-1822/1/artifact/docs/build/latex/appdev-environment-one-node-3TPs-kube.pdf

File: docs/source/app_developers_guide/kubernetes.rst [hunk 3]
```diff
@@ -0,0 +1,677 @@
...
+  Sawtooth functionality. The ``sawtooth-intkey-tp-python`` transaction
+  processor works with the ``int-key`` client, which has shell commands to
+  perform integer-based transactions.
+
+* `XO <../transaction_family_specifications/xo_transaction_family>`_
+  is a simple application for playing a game of tic-tac-toe on the blockchain.
+  The ``sawtooth-xo-tp-python`` transaction processor works with the ``xo``
+  client, which has shell commands to define players and play a game.
+  XO is described in a later tutorial.
+
```
[the remark this answers is not in the document]
Comment (chenette) [reply docs/source/app_developers_guide/kubernetes.rst]: Found it! There is an extra space on the next line. Sphinx is very fussy about spaces.

**Comment by chenette** [comment]
I pushed a fixup commit with @rberg2's suggested changes, plus an alignment tweak for the diagram.

**Comment by chenette** [comment]
I pushed a fixup commit to fix the broken links. I also corrected a few other things, such as a typo, the name of the kubeconfig YAML file, and example prompts (made them consistent with the upcoming multi-node procedure).

**Comment by chenette** [comment]
I can't link to the kubeconfig YAML file, because the file isn't there yet (it's being added in a separate PR).  When that PR has been merged, I'll add the link in the Kubernetes procedure.

**Comment by chenette** [comment]
Added a commit that links to the new Kubernetes YAML file (from PR #1830 ).

**Pull request merged**